gnosis/safe/proxy_factory.py

```diff
@@ -57,6 +66,36 @@ def deploy_contract(
             deployer_account, constructor_data
         )
 
+    @cache
+    def get_proxy_creation_code(self) -> bytes:
+        """
+        :return: Creation code used for the Proxy deployment.
+            With this it is easily possible to calculate predicted address.
+        """
+        return self.contract.functions.proxyCreationCode().call()
+
+    @cache
+    def get_proxy_runtime_code(self) -> Optional[bytes]:
+        """
+        :return: Runtime code of a deployed Proxy. For v1.4.1 onwards the method is not available, so `None`
```

Comment:
For 1.4.1 the method was deprecated? If yes why not overwrite the function on 1.4.1 and raise deprecated exception?

Reply:
Because this is required for the `check_proxy` method to work, we would need to refactor `check_proxy_code`. Also this was present on the previous version and it only shows as I sorted the methods to clean up the class